### PR TITLE
worked on Deploying a SAC within a contract.

### PR DESCRIPTION
**Pull request by KoxyG**

### CODE REVIEW

**Comment by briwylde08** [comment]
https://github.com/stellar/stellar-docs/issues/625

**Comment by KoxyG** [comment]
Hi @briwylde08 any issues with reviewing my pr?

**Comment by briwylde08** [comment]
No issues, I'm getting everything assigned today. Reviewers have a couple of weeks to get to them!

File: docs/build/guides/conventions/deploying-a-SAC.mdx [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+
+## 2. Testing the deployment
+
+You need to test the deployment to ensure everything works as expected. The following code demonstrates how to test the SacDeployer contract using the Soroban Rust SDK's test utilities.
+
+```rust
+#[cfg(feature = "testutils")]
+fn main() {
+    let env = TestEnv::default();
+    let serialized_asset = Bytes::from("actual serialized asset"); // Replace with actual serialized asset info
```
[the remark this answers is not in the document]
Comment (KoxyG) [reply docs/build/guides/conventions/deploying-a-SAC.mdx:73]: I just made updated it, ps check @ElliotFriend  

https://github.com/stellar/stellar-docs/pull/902

[the remark this answers is not in the document]
Comment (ElliotFriend) [reply docs/build/guides/conventions/deploying-a-SAC.mdx:73]: I wasn't sure why you had made a new PR, so I tried to merge that one back into this one, but now we've got a bunch of extra files. (that's probably why you made the PR lol.)

I'll pick up the review over in #902 

Thanks!

**Comment by ElliotFriend** [comment]
Closing this PR and moving the conversations to #902

**Pull request closed without merging**